changelog/unreleased/update-reva-beta.5.md

```diff
@@ -0,0 +1,3 @@
+Enhancement: Update reva
+
+https://github.com/owncloud/ocis/pull/4114
```

Comment:
```suggestion
https://github.com/owncloud/ocis/pull/4115
```

Reply:
weird 4114 did not show up in https://github.com/owncloud/ocis/issues?q=